.github/workflows/main.yml

```diff
@@ -48,13 +64,27 @@ jobs:
       uses: actions/setup-dotnet@v2
       with:
         dotnet-version: '6.0.x'
+      
+    # Adjustment is done prior to Nuke build as OCTOVERSION information is included in the result package.
+    - name: Append OCTOVERSION_CurrentBranch with -nightly (for scheduled)
+      if: github.event_name == 'schedule'
+      run: echo "OCTOVERSION_CurrentBranch=${{ env.OCTOVERSION_CurrentBranch }}-nightly" >> $GITHUB_ENV
```

Comment:
For scheduled event, expecting `OCTOVERSION_CurrentBranch` = `refs/heads/main`
Adjustment = `refs/heads/main-nightly`
Resulting `OCTOVERSION_FullSemVer` major.minor.patch`-main-nightly`

Reply:
Having `main` show in versions is atypical, given it's the default branch.

Is there a reason to recalculate the existing branch name? Above it's already set to `${{ github.head_ref || github.ref }}`. Would it be safer to simply append `-nightly` to whatever the current value happens to be?

Reply:
Ok, sorry I understand now. I was confused by `-main` being in the pre-release pacakge name. This does indeed make sense, as `OctoVersion` now sees the branch name as `main-nightly`, which will not match its conditions for building a release version without the branch name in the pre-release tag. My mistake!